archivy/data.py

```diff
@@ -7,10 +7,12 @@
 from shutil import rmtree
 
 import frontmatter
+from flask import current_app
 
-from archivy.config import Config
 
-DIRNAME = Config.APP_PATH + "/data/"
+# FIXME: ugly hack to make sure the app path is evaluated at the right time
+def get_data_dir():
+    return os.path.join(current_app.config['APP_PATH'], "data")
```

Comment:
This needs to be:
```
     return os.path.join(current_app.config['APP_PATH'], "data/")```

 trailing slash otherwise the paths are not parsed correctly.